gateway/services/upload.go

```diff
@@ -49,6 +50,12 @@ var UploadRoutes = arbor.RouteCollection{
 		"/upload/blobstore/{id}/",
 		alice.New(middleware.IdentificationMiddleware).ThenFunc(GetBlob).ServeHTTP,
 	},
+	arbor.Route{
+		"DeleteBlob",
+		"DELETE",
+		"/upload/blobstore/{name}/", // ?? Is it correct to use {name}?
```

Comment:
`name` should work fine, but I would go with `id` since that's what we use elsewhere in this endpoint and what you use in the controller code

Reply:
Okay, I have changed it to id.